### PR TITLE
fix: Allow optional email and telephone for applicants

### PR DESCRIPTION
**Pull request by DafyddLlyr**
A recent Uniform submission has failed as applicant contact details were missing (email and telephone).

Our LDC service allows this, but the OneApp schema (and our Zod schema which represents it) does not.

This change allows applicant contact details to be optional, as long as they are supplied for an agent. I'm not totally sure if Uniform is validating this - I suspect it's not an issue but just something that might still cause a failure following this change.

### CODE REVIEW

File: src/export/oneApp/OneApp.test.ts
Comment (jessicamcinchak) [inline src/export/oneApp/OneApp.test.ts:865]: thanks for clear tests :+1:

**Review by jessicamcinchak — APPROVED** [review]

**Comment by DafyddLlyr** [comment]
Holding off for content confirmation from Alastair and/or George before I merge this one, juuust in case my assumptions are incorrect.

**Pull request merged**
